demos/common/cpp/models/src/image_model.cpp

```diff
@@ -45,7 +45,9 @@ std::shared_ptr<InternalModelData> ImageModel::preprocess(const InputData& input
         const size_t height = tensorShape[ov::layout::height_idx(layout)];
         const size_t channels = tensorShape[ov::layout::channels_idx(layout)];
         if (static_cast<size_t>(img.channels()) != channels) {
-            throw std::runtime_error("The number of channels for model input and image must match");
+            throw std::runtime_error(std::string("The number of channels for model input: ") +
+                                     std::to_string(channels) + " and image: " +
+                                     std::to_string(static_cast<size_t>(img.channels())) + " - must match");
```

Comment:
```suggestion
                                     std::to_string(img.channels()) + " - must match");
```
Can't `to_string()` construct from int?

Reply:
fair enough, applied